src/config/pages/transaction-list.json

```diff
@@ -22,16 +22,14 @@
                     "deadline",
                     "transactionHash",
                     "transactionDescriptor",
-                    "maxFee",
```

Comment:
any reason remove the maxfee from the list?

Reply:
`maxFee` is not a criteria to find transaction in the list in most cases. In desktop view in my opinion it is better to show `value` instead